explainability-service/src/main/java/org/kie/trustyai/service/data/storage/MemoryStorage.java

```diff
@@ -10,17 +10,19 @@
 
 import javax.enterprise.context.ApplicationScoped;
 
+import org.jboss.logging.Logger;
```

Comment:
Minor note, it should be better to use `slfj4-api` to be implementation agnostic.

Reply:
👍 https://issues.redhat.com/browse/FAI-972